tenzir/tests/exec/operators/from/invalid_scheme.tql

```diff
@@ -0,0 +1,4 @@
+// error: true
+// skip: macos
```

Comment:
Why change the semantic of `skip: <reason>`?

Reply:
I wasn't aware that `<reason>` was supported yesterday. I found only one instance where it was actually used, and I needed a way to skip a test only on macOS (because the diagnostic is different).

Reply:
The options are all documented in the README.md file in the same directory.

Reply:
The problem is still there: Some tests produce different output depending on the build configuration. This is the case in our CI, where the macOS build does not have abfs and gcs support.

Ideally the tests should not be sensitive to this config at all, but we also don't have any mechanism to post-process the diagnostic, or disable the hinting at the node config or TQL level.

I'd favor any of these other approaches, but for a single instance this feels over-engineered.
In any case, I think the best option for now is to drop the new test and extend `run.py` when we have agreement about what feature we want for this.

Reply:
Arguably it'd be better to enable the ABFS and GCS plugins on macOS, but to have them error at runtime saying that they're not supported on the platform. This'd solve the issue as far as I can tell.